### PR TITLE
Add Enzyme deps and configuration

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

**Comment by ohadlevy** [comment]
:+1: can you add a basic test? (e.g for App.js) that we can see snapshots are created correctly?

File: package.json [hunk 1]
```diff
@@ -113,5 +114,9 @@
     "presets": [
       "react-app"
     ]
+  },
+  "devDependencies": {
+    "enzyme": "^3.5.1",
+    "enzyme-adapter-react-16": "^1.3.1"
```
[the remark this answers is not in the document]
Comment (rawagner) [reply package.json]: I do not have any experience with enzyme, but looking at what `enzyme-to-json` is doing, Im pretty sure we need it :)

**Comment by rawagner** [comment]
@ohadlevy can you please take a look again ?

**Comment by ohadlevy** [comment]
thanks :)

**Pull request merged**